src/resolve/resolved-dns-packet.h

```diff
@@ -253,6 +253,8 @@ int dns_packet_extract(DnsPacket *p);
 
 bool dns_packet_equal(const DnsPacket *a, const DnsPacket *b);
 
+int dns_packet_ede_rcode(DnsPacket *p, uint16_t *ret_ede_rcode);
+bool ede_rcode_is_dnssec(int ede_rcode);
```

Comment:
we generally try to prefix our functions that deal with protocol details with "dns_…", please do so here too